terminus-ui/chip/src/chip-collection.component.ts

```diff
@@ -491,30 +481,23 @@ export class TsChipCollectionComponent implements AfterViewInit, OnInit, OnDestr
     }
   }
 
-  public clickSelect(clickEvent) {
-    const shiftKey = clickEvent.event.shiftKey;
-    if (this.allowMultipleSelections && shiftKey) {
-      // Disable text highlight when shift click
-      document.onselectstart = function() {
-        return false;
-      };
-      clickEvent.chip.toggleSelected();
-      clickEvent.event.preventDefault();
-    }
```

Comment:
Does shift + click work w/o this?
Oh I see you moved this to chip component....